generator/src/test/java/com/scottlogic/deg/generator/cucumber/steps/NumericValueStep.java

```diff
@@ -34,12 +35,45 @@ public void producedDataShouldContainNumericValuesForField(String fieldName){
         helper.assertFieldContainsNullOrMatching(fieldName, Number.class);
     }
 
+    @Then("{fieldVar} contains anything but numeric data")
+    public void producedDataShouldContainAnythingButStringValuesForField(String fieldName){
+        helper.assertFieldContainsNullOrNotMatching(fieldName, Number.class);
+    }
+
     @Then("{fieldVar} contains numeric values between {number} and {number} inclusively")
     public void producedDataShouldContainNumericValuesInRangeForField(String fieldName, Number minInclusive, Number maxInclusive){
         helper.assertFieldContainsNullOrMatching(
             fieldName,
             Number.class,
-            value -> isGreaterThanOrEqual(value, minInclusive) && isLessThanOrEqual(value, maxInclusive));
+            isBetweenInclusively(minInclusive, maxInclusive));
+    }
+
+    @Then("{fieldVar} contains numeric values less than or equal to {number}")
+    public void producedDataShouldContainNumericValuesLessThanForField(String fieldName, Number lessThanInclusive){
+        helper.assertFieldContainsNullOrMatching(
+            fieldName,
+            Number.class,
+            value -> isLessThanOrEqual(value, lessThanInclusive));
+    }
+
+    @Then("{fieldVar} contains numeric values greater than or equal to {number}")
+    public void producedDataShouldContainNumericValuesGreaterThanForField(String fieldName, Number greaterThanInclusive){
+        helper.assertFieldContainsNullOrMatching(
+            fieldName,
+            Number.class,
+            value -> isGreaterThanOrEqual(value, greaterThanInclusive));
+    }
+
+    @Then("{fieldVar} contains numeric values outside {number} and {number}")
```

Comment:
Do we need to add `inclusively` to this description?

Reply:
No, agreed to treat this as a full complement of between (which is inclusive) therefore this one should be exclusive